### DISCUSSION TITLE
Termux removing trailing character in command execution

### DISCUSSION THREAD

**Discussion by GuillaumeMilan** [termux/termux-app#2542] [General]
I am running the termux version 0.117
When I execute a command, the trailing whitespace before the current cursor, if and only if it was written as the last character. 

For instance (I use the `¦` for cursor position  in the following examples):

Let's get the following command example: (I use the `¦` char as the cursor pointer)

I got the following command to modify

```
mix my_task
```

So I go after `mix` and type `help` to result to the following command

```
mix help ¦my_task
```

And termux tries to execute

```
mix helpmy_task
```

Should I create an issue for this?

**Comment by agnostic-apollo** [comment]
Try adding `enforce-char-based-input=true` to `~/.termux/termux.properties`

https://wiki.termux.com/wiki/Terminal_Settings#Workaround_for_some_keyboards_issues

Also check related existing issue #2421
